### PR TITLE
Improve ViewRenderer

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️/❌
| Breaks BC?    | ❌
| Fixed issues  | Unify finding layout with main package

### CODE REVIEW

File: src/ViewRenderer.php [hunk 1]
```diff
@@ -75,7 +75,7 @@ public function __construct(
      */
     public function getViewPath(): string
     {
-        return $this->aliases->get($this->viewPath) . ($this->name ? '/' . $this->name : '');
+        return $this->aliases->get($this->viewPath);
```
Comment (vjik) [inline src/ViewRenderer.php:78]: Why? Seems, it's return incorrect view path without name.

File: src/ViewRenderer.php [hunk 2]
```diff
@@ -498,7 +500,13 @@ private function findLayoutFile(string $file, WebView $view): string
             return $file;
         }
 
-        return $file . '.' . $view->getDefaultExtension();
+        $layoutFile = $file . '.' . $view->getDefaultExtension();
+
+        if ($view->getDefaultExtension() !== 'php' && !is_file($layoutFile)) {
+            $layoutFile = $file . '.php';
+        }
```
Comment (vjik) [inline src/ViewRenderer.php:507]: Why need it?

Comment (rustamwin) [reply src/ViewRenderer.php:507]: It's needed when layout is php template and other views is different template e.g twig. See https://github.com/yiisoft/view/blob/f378b696917c2a086b634ef072ccd19b16859f88/src/ViewTrait.php#L615-L619

File: src/ViewRenderer.php (hunk 1 above)
Comment (rustamwin) [reply src/ViewRenderer.php:78]: It may confuses with scoped view.

Comment (rustamwin) [reply src/ViewRenderer.php]: Reverted.

**Comment by samdark** [comment]
Need a line for CHANGELOG.

**Review by samdark — APPROVED** [review]

**Pull request merged**